docs/framework/migration-guide/retargeting/obsoleteattribute_exports_as_both_obsoleteattribute_and_deprecatedattribute_in_winmd_scenarios.md

```diff
@@ -0,0 +1,10 @@
+### ObsoleteAttribute exports as both ObsoleteAttribute and DeprecatedAttribute in WinMD scenarios
+
+|   |   |
+|---|---|
+|Details|When you create a Windows Metadata library (.winmd file), the <xref:System.ObsoleteAttribute?displayProperty=name> attribute is exported as both <xref:System.ObsoleteAttribute?displayProperty=name> and Windows.Foundation.DeprecatedAttribute.|
```

Comment:
should we link to the API in docs? https://docs.microsoft.com/en-us/uwp/api/windows.foundation.metadata.deprecatedattribute